README.md

```diff
@@ -67,3 +67,9 @@ That will add all the boilerplate and scripts to your project
 # Inspired by
 
 Dan Abramov - The Melting Pot of JavaScript : https://www.youtube.com/watch?v=G39lKaONAlA
+
+# Changeset
+
+## 1.0.0
+
+All the dependencies are bumped to the latest major version. 
```

Comment:
@zzarcon not sure, but it would be good for consumer to know what have we bumped?

Reply:
It would be nice, but consumers of this really shouldn't care, as long as it works. That's the point of this lib, to hide all those annoying tooling configs and version, etc

Reply:
as we discussed their tests are dependant on version of jest we provide. Or their TS code is dependand on type of TS we provide.